pkg/netutil/netutil.go

```diff
@@ -48,66 +48,69 @@ type CNIEnv struct {
 }
 
 func DefaultConfigList(e *CNIEnv) (*NetworkConfigList, error) {
-	return GenerateConfigList(e, nil, DefaultID, DefaultNetworkName, DefaultCIDR)
+	ipam, _ := GenerateIPAM("", DefaultCIDR)
+	plugins, _ := GenerateCNIPlugins("", DefaultID, ipam)
+	return GenerateConfigList(e, nil, DefaultID, DefaultNetworkName, plugins)
 }
 
+const ConfigListTemplate = `{
+  "cniVersion": "0.4.0",
+  "name": "{{.Name}}",
+  "nerdctlID": {{.ID}},
+  "nerdctlLabels": {{.Labels}},
+  "plugins": {{.Plugins}}
+}`
+
```

Comment:
Can we just marshal a struct into JSON, without using text/template?

Reply:
SGTM, If you are not in a hurry, I may write another PR to implement.

Reply:
I'm back, rewrite it with marshal.